src/electron/views/left-nav/fluent-left-nav.tsx

```diff
@@ -37,11 +37,11 @@ export const FluentLeftNav = NamedFC<FluentLeftNavProps>('LeftNav', props => {
             layerProps={{ className: styles.leftNavPanelHost }}
             className={styles.leftNavPanel}
             isOpen={isNavOpen}
-            isLightDismiss
```

Comment:
Removing this line isn't technically necessary, but I think it makes it more clear (it's an no-op here because it's unilaterally overriden in the GenericPanel impl)